### PR TITLE
[15.0][MIG] chained_swapper: Migration to 15.0

### PR DESCRIPTION
**Pull request by cesar-tecnativa**
@Tecnativa TT36449

### CODE REVIEW

**Comment by cesar-tecnativa** [comment]
@pedrobaeza could you please check it?

File: chained_swapper/models/chained_swapper.py [hunk 1]
```diff
@@ -145,10 +146,18 @@ def _check_sub_field_chain(self):
...
                 chain_model[chain_field_name]  # pylint: disable=W0104
-            except KeyError:
+            except KeyError as err:
                 raise exceptions.ValidationError(
-                    _("Incorrect sub-field expression:") + " " + rec.sub_field_chain
-                )
+                    _("Incorrect sub-field expression: %(sub_field_chain)s. %(error)s")
+                    % {"sub_field_chain": rec.sub_field_chain, "error": err}
+                ) from err
+
```
Comment (pedrobaeza) [inline chained_swapper/models/chained_swapper.py]: Don't add empty lines in the middle of a method

File: chained_swapper/models/chained_swapper.py [hunk 2]
```diff
@@ -145,10 +146,18 @@ def _check_sub_field_chain(self):
...
+            except Exception as err:
```
Comment (pedrobaeza) [inline chained_swapper/models/chained_swapper.py:154]: Why are you adding this catch?

Comment (cesar-tecnativa) [reply chained_swapper/models/chained_swapper.py:154]: On line 

145: chain_model = self.env[rec.chained_swapper_id.model_id.model]

rec.chained_swapper_id can be Null -> therefore an exception.

**Comment by cesar-tecnativa** [comment]
@chienandalu could you please review this?

**Review by chienandalu — APPROVED** [review]
Tested :+1:

File: chained_swapper/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Chained Swapper",
+    "summary": "Chained Swapper",
+    "version": "15.0.0.1.0",
```
Comment (pedrobaeza) [inline chained_swapper/__manifest__.py]:
```suggestion
    "version": "15.0.1.0.0",
```

**Pull request merged**